src/config/configuration.ts

```diff
@@ -0,0 +1,11 @@
+import { config } from 'dotenv';
+import Iconfig from './IConfig';
+
+config();
+
+const configuration: Iconfig = Object.freeze({
+port: process.env.PORT,
+env: process.env.NODE_ENV
+});
```

Comment:
Why have we wrapped this configuration in the Object.freeze method

Reply:
Object.freeze is used so that no new properties could be added, and no alteration could be done to the existing data in the object.